frontend/src/questions/components/LabelIconPicker.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component, PropTypes } from "react";
+
+import S from "./LabelIconPicker.css";
```

Comment:
I would vote against single character names like this.  I know that right now for you it's obvious what this means and the brevity is cool, but in the long run it's a lot better when the code reads easily without needing prior knowledge.  Maybe just `CSS`?  which is only a couple characters longer.